cli/src/template.rs

```diff
@@ -184,6 +184,35 @@ describe('{}', () => {{
     )
 }
 
+pub fn package_json() -> &'static str {
+    r#"{
+    "dependencies": {
+        "@project-serum/anchor": "^0.13.2"
```

Comment:
It will be much better if this will be a template. `CARGO_PKG_VERSION` from environment variables can be used (https://doc.rust-lang.org/cargo/reference/environment-variables.html).

Reply:
Agreed lets make this change. The CLI uses this for other things already.

Reply:
Very good point done